test/integration/issue.2118.yaml

```
@@ -4,4 +4,4 @@ helmDefaults:
   timeout: 300
 releases:
 - name: argo-cd-crds
-  chart: git::https://github.com/argoproj/argo-helm.git@charts/argo-cd/crds?ref=main
+  chart: git::https://github.com/argoproj/argo-helm.git@charts/argo-cd/crds?ref=argocd-notifications-1.8.1
```

They seem to have moved to `charts/argo-cd/templates/crds` 16 hours ago:

https://github.com/argoproj/argo-helm/commit/e6587671d38484d6de61dae033d56340361976f5

We want the vanilla CRD manifests but what they provide in the sub-directory within the chart is now Helm templates so...

How about changing the reference to `https://github.com/argoproj/argo-cd/tree/master/manifests/crds`?

```suggestion
  chart: git::https://github.com/argoproj/argo-cd.git@manifests/crds?ref=main
```

From what I can see from the contents of the directory, it would work

https://github.com/argoproj/argo-cd/tree/master/manifests/crds